source/wp-content/themes/wporg-developer-2023/inc/cli.php

```diff
@@ -207,8 +207,12 @@ private static function update_post_from_markdown_source( $post_id ) {
 		if ( is_wp_error( $markdown_source ) ) {
 			return $markdown_source;
 		}
-		if ( ! function_exists( 'jetpack_require_lib' ) ) {
-			return new WP_Error( 'missing-jetpack-require-lib', 'jetpack_require_lib() is missing on system.' );
+		
+		if ( ! class_exists( 'WPCom_GHF_Markdown_Parser' ) && defined( 'JETPACK__PLUGIN_DIR' ) ) {
```

Comment:
I can help feel that combining the two `if ( ! class_exists( 'WPCom_GHF_Markdown_Parser' )` would be better here, particularly to avoid any future issue whereby we either do or don't need to check that class_exists statement.

Reply:
Maybe something like this:
```
// Comment saying why we are doing this.
if ( ! class_exists( 'WPCom_GHF_Markdown_Parser' ) {
    if ( defined( 'JETPACK__PLUGIN_DIR' ) ) {
        include JETPACK__PLUGIN_DIR . '/_inc/lib/markdown.php';
    } else {
        return new WP_Error( 'missing-jetpack-markdown', 'Jetpack Markdown is missing on system.' );
    }
}
```

Reply:
Updated 🙂